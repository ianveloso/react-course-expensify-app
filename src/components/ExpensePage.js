import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpensesSummary';

const ExpensePage = () => (
    <div>
        This is from my dashboard component
        <ExpenseList />
        <ExpenseListFilters />
        <ExpensesSummary />
    </div>
);

export default ExpensePage;