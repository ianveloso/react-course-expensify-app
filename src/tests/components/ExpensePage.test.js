import React from 'react';
import { shallow } from 'enzyme';
import ExpensePage from '../../components/ExpensePage';

test('should render Expense Dashboard correctly', () => {
    const wrapper = shallow(<ExpensePage />);
    expect(wrapper).toMatchSnapshot();

    //expect(wrapper.find('h1').text()).toBe('Expensify');
    // const renderer = new ReactShallowRenderer(); 
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot(); 
});