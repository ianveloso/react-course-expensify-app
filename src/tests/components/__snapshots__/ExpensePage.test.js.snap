// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense Dashboard correctly 1`] = `
<div>
  This is from my dashboard component
  <Connect(ExpenseList) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpensesSummary) />
</div>
`;
